packages/engine/src/networking/systems/NetworkTransportState.tsx

```diff
@@ -0,0 +1,153 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import { defineAction, defineState, none, receiveActions } from '@etherealengine/hyperflux'
+import { matches } from '../../common/functions/MatchesUtils'
+import { Engine } from '../../ecs/classes/Engine'
+import { defineSystem } from '../../ecs/functions/SystemFunctions'
+import { UserID } from '../../schemas/user/user.schema'
+
+export class NetworkTransportActions {
+  static requestTransport = defineAction({
+    type: 'ee.engine.network.TRANSPORT_REQUEST_CREATE',
+    direction: matches.literals('send', 'recv'),
+    sctpCapabilities: matches.object
+  })
+
+  static requestTransportError = defineAction({
+    type: 'ee.engine.network.TRANSPORT_REQUEST_ERROR_CREATE',
+    error: matches.string,
+    direction: matches.literals('send', 'recv')
+  })
+
+  static transportCreated = defineAction({
+    type: 'ee.engine.network.TRANSPORT_CREATED',
+    transportID: matches.string,
+    direction: matches.literals('send', 'recv'),
+    sctpParameters: matches.object,
+    iceParameters: matches.object,
+    iceCandidates: matches.arrayOf(matches.object),
+    dtlsParameters: matches.object
+  })
+
+  static requestTransportConnect = defineAction({
+    type: 'ee.engine.network.TRANSPORT_REQUEST_CONNECT',
+    requestID: matches.string,
+    transportID: matches.string,
+    dtlsParameters: matches.object
+  })
+
+  static requestTransportConnectError = defineAction({
+    type: 'ee.engine.network.TRANSPORT_REQUEST_ERROR_CONNECT',
+    requestID: matches.string,
+    error: matches.string
+  })
+
+  static transportConnected = defineAction({
+    type: 'ee.engine.network.TRANSPORT_CONNECTED',
+    requestID: matches.string,
+    transportID: matches.string
+  })
+
+  static closeTransport = defineAction({
```

Comment:
Just wondering if this can be renamed to `transportClosed`/`transportClosing` to make it sound similar to other actions in this file.

Reply:
yeah can do